### ISSUE TITLE
Homework review - week01 margin weird value

### ISSUE THREAD

**Issue by jakub-c** [Hedayatsamadi123/hyf-html-css#12]
https://github.com/Hedayatsamadi123/hyf-html-css/blob/3fbf624be26669a95777c5b6de3d539c9565e832/week01/style.css#L89

was it supposed to be `px` instead of `%`

**Comment by Hedayatsamadi123** [comment]
Yes i change it to px , i used % instead of px because i thought it will be responsive.
